pkg/migration-tool/planner/planner.go

```diff
@@ -177,27 +180,34 @@ func (p *Plan) createSlab(mint, maxt int64) (reference *Slab, err error) {
 	if percent > 100 {
 		percent = 100
 	}
-	baseDescription := fmt.Sprintf("progress: %.3f%% | slab-%d time-range: %d mins | mint: %d | maxt: %d", percent, id, timeRangeInMinutes, mint/second, maxt/second)
-	reference = &Slab{
-		id:                    id,
-		pbarDescriptionPrefix: baseDescription,
-		pbar: progressbar.NewOptions(
-			numStepsWriter,
-			progressbar.OptionOnCompletion(func() {
-				_, _ = fmt.Fprint(os.Stdout, "\n")
-			}),
-		),
-		numStores: p.config.NumStores,
-		stores:    make([]store, p.config.NumStores),
-		mint:      mint,
-		maxt:      maxt,
-		pbarMux:   p.pbarMux,
-		plan:      p,
-	}
-	reference.initStores()
+	baseDescription := fmt.Sprintf("progress: %.2f%% | slab-%d time-range: %d mins | begin: %d | end: %d", percent, id, timeRangeInMinutes, mint/second, maxt/second)
+	if p.config.HumanReadableTime {
+		baseDescription = fmt.Sprintf("progress: %.2f%% | slab-%d time-range: %d mins | begin: %s | end: %s", percent, id, timeRangeInMinutes, timestamp.Time(mint), timestamp.Time(maxt))
```

Comment:
We should keep both (and use depending on the input) since if a user is using unix, then time range in unix is important to give him an idea when to start from, the next time. Vice-versa for human-readable.